From ddc504ea11ee9f16202be0aa5fb104bae1273643 Mon Sep 17 00:00:00 2001
From: Steven M Campbell <scampbell@scampbell.net>
Date: Sun, 2 May 2021 17:39:16 -0400
Subject: [PATCH] emerald-theme-manager is now compatible with newer versions
 of tar. The tar command now requires that the exclude directives come before
 the specification of files to archive.   This new ordering is compatible with
 older versions of tar as well making this an easy fix.

---
 themer/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/themer/main.c b/themer/main.c
index c936a64..9aad0b4 100644
--- a/themer/main.c
+++ b/themer/main.c
@@ -512,7 +512,7 @@ static void export_theme(gchar * file)
     at = g_shell_quote(fn);
     g_free(fn);
     fn = g_shell_quote(file);
-    ot = g_strdup_printf("tar -czf %s -C %s ./ --exclude=*~",fn,at);
+    ot = g_strdup_printf("tar -czf %s -C %s --exclude=*~ ./",fn,at);
     if (!g_spawn_command_line_sync(ot,NULL,NULL,&ex,NULL) ||
             (WIFEXITED(ex)&&WEXITSTATUS(ex)))
     {
